numpy/_core/tests/test_cython.py

```diff
@@ -22,7 +22,7 @@
     # other fixes in the 0.29 series that are needed even for earlier
     # Python versions.
     # Note: keep in sync with the one in pyproject.toml
-    required_version = "0.29.30"
+    required_version = "0.29.35"
```

Comment:
This brings the test into sync with `pyproject.toml`

Reply:
We require `>=3.0.6` there?

Reply:
Ahh, right. I was looking at the other version checks in the nearby tests. They all should match the `pyproject.toml` one.